public/css/icinga/widgets.less

```diff
@@ -66,6 +68,10 @@
 
   position: relative;
 
+  p {
+    margin-bottom: 0;
```

Comment:
@flourish86 There's no bottom margin even without this. Why is this necessary?

Reply:
Never mind, I bet you've fixed the bottom notifications without knowing I've done this already with ef7bdf807ff75533e91c82ecf47758809bb12e3b

Reply:
I guess that's true 😆